### ISSUE TITLE
GetFileAttributesEx Error

### ISSUE THREAD

**Issue by cs2mw** [kardianos/govendor#395]
in windows 10 command,
go get govendor and exec govendor -version is ok
but, under %GOPATH%/src, create project "testgovendor",and exec command "govendor init".
govendor is not ok,and expose error like this:

**Error:GetFileAttributesEx src:The system cannot find the file specified.**

**Comment by gouez** [comment]
I have the same problem。。

**Comment by phrozen** [comment]
I have the same problem, is your code inside OneDrive?

**Comment by mpandzo** [comment]
I get the same error in Windows 10 

```
GOROOT=D:\Programs\Go
GOPATH=D:\proj
Path variable contains D:\proj\bin and D:\Programs\Go\bin
```

When I go to D:\proj\src\my-proj via cmd or gitbash and I run govendor init I get the following error:

```
$ govendor init
Error: GetFileAttributesEx src: The system cannot find the file specified.
```

**Comment by OuttaLineNomad** [comment]
I have the same issue. I changed my GOPATH from C:/Users/MyUser/Go  to C:/Users/Dropbox/Projects/go. I deleted the old path. 

When I run govendor init with a project in Dropbox/projects/go/src I get this error.
``
$ govendor init
Error: GetFileAttributesEx src: The system cannot find the file specified.
``
I have no idea what else to do. I also deleted govendor cache also deleted and reinstalled govendor. Does anyone have a fix? Or a way I can get more info on this error.
